demo/core_10_base.js

```diff
@@ -387,7 +387,19 @@ $.user.eval = function($$$cmd) {
       $$$cmd = 'Unhandled exception: ' + String(e);
     }
   }
-  user.narrate($$$cmd);
+  // Save the value without creating a new variable.
+  $.user.eval.value_ = $$$cmd;
```

Comment:
Maybe this whole function should be restructured as:

```JS
  try {
    (function (result) { /* report normal eval result */ })(eval($$$cmd));
  } catch (e) {
    (function (exception) { /* report abnormal eval result */ })(e);
  }

(This could also allow the result message to more clearly distinguish between a *thrown* Error value and a *returned* one.)